PULL_REQUEST_MANAGEMENT.md

```diff
@@ -0,0 +1,14 @@
+# Rules for managing Pull Requests
+
+For the sake of trying to maintain an acceptable number of open but idle PRs, the following rules should be considered:
+- If a PR has at least one approval, it can be merged after 1 week of waiting for additional comments.
+  - If you consider the change trivial enough or it has at least 3 approvals, it may also be merged right away.
+- If a PR stays in "changes requested" for 2 weeks, and that change has not been done in the meantime, it shall be closed.
```

Comment:
What if this change requires more than 2 weeks of time for that person?

Reply:
I'd change it to something like “...and there is no reaction from authors side...“

Reply:
I would say if there is "changes requested" give them 3 weeks then if there was no changes made, give them 2 more weeks for them to respond about their situation "Waiting for response". Then if they don't respond then close the PR.

Reply:
It should also be closed with a note to say "feel free to reopen after this date if you have additional changes or comments"

Reply:
I'm not sure whether contributors outside ReactOS organization are able to reopen PRs if they were closed by members.

Reply:
That sounds like a potential issue. Do we advise they open a new PR in this case?

Reply:
Yep, they can comment and ask to reopen, unless we use the "Lock conversation" link to prevent external users from participating, if the need arises.